dev/cf_cli/cf_cli.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+set -o errexit -o nounset
+
+namespace="scf"
+deployment_name="scf"
+pod_name="cf-terminal"
+api_ip=""
+echo "Waiting for endpoint..."
+while true; do
+  api_ip=$(kubectl describe endpoints -n "${namespace}" "${deployment_name}-router" | awk 'match($0, /  Addresses:[ ]+(.*)/, ip) { print ip[1] }')
+  if [ -n "${api_ip}" ]; then break; fi
+  sleep 3
+done
```

Comment:
Does it make sense to show the chosen IP ? (Maybe also endpoint or service, to know if router direct, or ingress is used).

Reply:
Good suggestion.